src/main/scala/faces/utils/Helpers.scala

```diff
@@ -214,6 +214,54 @@ case class Helpers(cfg: FacesSettings)(implicit rnd: Random) {
 
   }
 
+  // center the face in a way that simulates a face detection result (including uncertainty);
```

Comment:
Can you mention which detectors or which datasets are most compatible with this setting?